doc/_static/css/custom.css

```diff
@@ -233,17 +233,104 @@ div.sk-landing-footer {
 }
 
 /** Scikit-learn buttons ***************************/
-a.sk-btn-orange {
+/* a.sk-btn-orange {
     font-size: 1.1rem;
     font-weight: 500;
-    background-color: #f99f44;  /* sk-orange-tint-1 */
+    background-color: #f99f44;  // sk-orange-tint-1
     color: black !important;
+} */
+
+a.sk-btn-orange {
+    font-size: 1.1rem;
+    font-weight: 500;
+    background-color: #f99f44; /* Original sk-orange-tint-1 */
+    color: #ffffff; /* Changed to white for better contrast */
+    border: 2px solid #f99f44; /* Border to match background color */
+    padding: 10px 20px; /* Added padding for better button size */
+    border-radius: 5px; /* Rounded corners for a modern look */
+    text-decoration: none; /* Remove underline */
+    transition: background-color 0.3s ease, color 0.3s ease; /* Smooth transition */
 }
 
 a.sk-btn-orange:hover {
-    background-color: #fcb575;  /* --sk-orange-tint-3 */
+    background-color: #ffffff; /* Fill with white on hover */
+    color: #f99f44; /* Text color changes to match the original background color */
+    border: 2px solid #f99f44; /* Border remains the same on hover */
+}
+
+
+a.sk-btn-orange:hover {
+    background-color: #fbe1ca;  /* --sk-orange-tint-3 */
 }
 
+
+.container {
+    max-width: 80%; /* Restrict container width for better readability */
```

Comment:
on narrow screens we don't want to waste 20% of the screen width. So I would suggest setting that in size rather than percentage eg max-width: 180ch;